api_reference/source/_static/custom.css

```diff
@@ -0,0 +1,7 @@
+.navbar-nav .nav-link {
+    text-transform: capitalize;
+}
+
+.ethical-sidebar {
+    display: none;
+}
```

Comment:
Just making sure this doesn't get us in trouble with readthedocs or is this common practice?

Reply:
This addresses an issue where the sidebar ad would display twice instead of once. Already tested out and doesn't cause issues with the ad displaying just once on pages or getting flagged by readthedocs. Will most likely remove in the future.